detector/aws_instance_default_standard_volume.go

```diff
@@ -7,20 +7,17 @@ import (
 
 type AwsInstanceDefaultStandardVolumeDetector struct {
 	*Detector
-	IssueType  string
-	TargetType string
-	Target     string
-	DeepCheck  bool
 }
 
 func (d *Detector) CreateAwsInstanceDefaultStandardVolumeDetector() *AwsInstanceDefaultStandardVolumeDetector {
-	return &AwsInstanceDefaultStandardVolumeDetector{
-		Detector:   d,
-		IssueType:  issue.WARNING,
-		TargetType: "resource",
-		Target:     "aws_instance",
-		DeepCheck:  false,
-	}
+	nd := &AwsInstanceDefaultStandardVolumeDetector{Detector: d}
+	nd.Name = "aws_instance_default_standard_volume"
+	nd.IssueType = issue.WARNING
+	nd.TargetType = "resource"
+	nd.Target = "aws_instance"
```

Comment:
__[gometalinter]__
* [goconst] 8 other occurrence(s) of "aws_instance" found in: aws_instance_invalid_type.go:23:14 aws_instance_not_specified_iam_profile.go:17:14 aws_instance_invalid_subnet.go:23:14 aws_instance_previous_type.go:23:14 aws_instance_invalid_key_name.go:23:14 aws_instance_invalid_ami.go:23:14 aws_instance_invalid_vpc_security_group.go:24:14 aws_instance_invalid_iam_profile.go:23:14

<!-- Comment by [SideCI](https://sideci.com) -->